specs/signing-and-verification-workflow.md

```diff
@@ -82,11 +82,13 @@ The user wants to sign an arbitrary blob with a detached signature.
 ### Signing Steps
 
 1. **Generate signature:** Using notation CLI or any other compliant signing tool, sign the blob. The signing tool should follow the following guideline.
-    1. Construct the blob payload as defined in [`signature specification`](./signature-specification.md#payload)
+    1. Construct the blob payload:
+       - For COSE format, only [`COSE Hash Envelope`](./signature-envelope-cose.md/#cose-hash-envelope) is supported when signing arbitrary blobs.
```

Comment:
@yizha1 @FeynmanZhou Is this what we agree with?

Reply:
I see that there are `cose` and `cose-hash`.

Reply:
Basically, we generate payload depending on which envelope type we choose.

Reply:
@shizhMSFT  I believe we only need cose-hash for blob signature.